HOWTO.md

```diff
@@ -0,0 +1,109 @@
+# Changes in recent Tigervnc versions
+Previous Tigervnc versions had a wrapper script called `vncserver`. This script
+could be run as a user manually to start *Xvnc* process. The usage was quite 
+simple as you just run:
+```
+$ vncserver :x [vncserver options] [Xvnc options]
+```
+and that was it. It worked just fine, however, there were issues when users
```

Comment:
It worked fine for some cases, but far from all. So I think we should use more cautious wording here.